proposals/002_Summary.md

```diff
@@ -0,0 +1,109 @@
+
+## Motivation
+
+There are few motivations for this:
+
+ * A client should have a way to get the formal schema for items in a collection if one exists.  The current suggestion to use a link with `rel="describedBy"` doesn't fully address this need since it doesn't provide a robust way for a client to get the schema for multiple content types (for example, if a service supports GML, KML, and GeoJSON output, the `type` and `rel` of a link is not enough information for a client to know which schema applies to which output format).
+
+ * A "filter" extension should have a way to describe which properties are filterable and how clients should access a specific property in a filter expression.
+
+ * A "search" extension should have a way to describe which properties can be used in sorting.
+
+ * An "editing" extension should have a way to describe which properties are writeable (e.g. in an update) or might want to allow for a different schema for each method (e.g. one schema for create, one for update).
+
+## Proposed solution
+
+## Example
+
+Here is an example of what this summary resource might look like for a "buildings" collection (e.g. at https://example.com/collections/buildings/summary)
+
+```json
+{
+  "properties": {
+    "application/geo+json": {
+      "height": {
+        "schema": {
+          "href": "http://example.com/collections/buildings/schema#/definitions/height"
+        }
+      },
+      "age": {
+        "schema": {
+          "href": "http://example.com/collections/foo/schema#/definitions/age"
+        },
+        "filter": {
+          "allowed": false
+        },
+        "search": {
+          "sortable": false
+        }
+      },
+      "taxId": {
+        "schema": {
+          "href": "http://example.com/collections/foo/schema#/definitions/taxId"
+        },
+        "editing": {
+          "writeable": false
+        }
+      }
+    }
+  }
+}
+```
+
+Notes on the above example:
+
+ * The properties of the `properties` object (ack!) are content types.  Using an object means that there will be one member per content type.  So it looks like the responses `content` object in OpenAPI 3.
+
+ * The `schema` property would be defined by the core Features spec.  In the case of JSON schema, an href could deeply link to the spceific definition.  In the case of an XSD doc, there could be two properties: `href` and `path` (maybe?) so an XPath expression could be provided to refer to the part of the schema specific to a single property.
+
+ * Individual extensions could add their own members to each of the properties object.  For example, the "filter" expression could define a `filter` object.
+ 
+## Detailed design
+
+TODO
+
+## Backwards compatibility
+
+This proposal doesn't introduce any breaking changes.
+
+## Alternatives considered
```

Comment:
Have you had a look at the STAC Collection summaries? Even if you not consider it, it would be good if names would not conflict too much...